src/main/groovy/com/ullink/Ikvm.groovy

```diff
@@ -123,7 +123,12 @@ class Ikvm extends ConventionTask {
             return project.file(getKeyFile())
         }
         return new File[0]
-    } 
+    }
+
+    @InputFiles
+    def getJar() {
```

Comment:
maybe you can also reuse this getter below in build method instead of archivePath (for consistency) ?